core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -232,10 +232,10 @@ private[op] class OpMultiClassificationEvaluator
     ThresholdMetrics(
       topNs = topNs,
       thresholds = thresholds,
-      correctCounts = agg.mapValues { case (cor, _) => cor.toSeq },
```

Comment:
nit: just for completeness
one can materialize mapValues view by 
```scala
agg.mapValues { case (cor, _) => cor.toSeq }.view.toMap
```
or 
```scala
agg.transform { case (_, (cor, _)) => cor.toSeq },
```

Reply:
Good point. Thanks.